README.md

```diff
@@ -31,31 +29,52 @@ It is possible to customize the generated PDF using the following parameters:
     * Landscape
 * Page dimensions
 * Page size
-    * A4
     * Letter
-* Zoom
-* Quality
+    * Legal
+    * Tabloid
+    * Ledger
+    * A0
+    * A1
+    * A2
+    * A3
+    * A4
+    * A5
+    * A6
+* Scale
 * Margins
-* Minimum font size
-* No background
-* Grayscale
-* Enable/Disable smart shrinking
+* With background
+* Custom header and footer
 
 ### Setting parameters ###
 
-The following call will set portrait orientation, left margin, right margin and grayscale:
+The following call will set portrait page format, orientation, left margin, right margin and document background:
 ```java
 Document.fromStaticHtml("<html>")
         .and()
         .withParameters()
+            .pageFormat("A5")
             .portrait()
             .marginLeft("2cm")
             .marginRight("2cm")
-            .grayscale()
+            .withBackground()
```

Comment:
this looks ambigous to me. What kind of background should I expect? Color? Image? If color, then which one? Can I customeize it anyhow? Can we introduce more meaningful name of this method?